src/kolibri_android/main_activity/activity.py

```diff
@@ -145,7 +164,7 @@ def on_activity_save_instance_state(self, activity, out_state_bundle):
         logging.info(f"Saved Kolibri path: '{kolibri_path or ''}'")
 
     def _get_current_kolibri_path(self):
-        current_url = PythonActivity.mWebView.getUrl()
+        current_url = FullScreen.getInstance().getUrl()
```

Comment:
Nit: Maybe is a good time to rename "FullScreen" to a better name?

Reply:
Okay, after some deliberation I renamed it to `KolibriAndroidHelper`. Not a great name, but at least it isn't misleading :)